cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -119,6 +119,14 @@ the argument will be set for all programs that use ``myTable``. In order to set
 program ``myMR``, the dataset argument can be nested inside the program scope as
 ``mapreduce.myMR.dataset.myTable.cache.seconds=30``.
 
+Overriding Logical Start Time with Runtime Arguments
+====================================================
+
+The runtime argument ``logicalStartTime`` can be used to override the logical start time of a Workflow, MapReduce,
```

Comment:
we do not have a true standard for naming. However, it seems that dot notation is preferred, and in this case, shouldn't it be logical.start.time, and could be optionally prefixed as workflow.<name>.logical.start.time, or mapreduce.<name>.logical.start.time. 

Another question I have, today the context has a method getLogicalStartTime(). If we support this as a runtime argument, does that mean that alternatively, context.getRuntimeArguments.get("logical.start.time") can be used?